test/test-utils.test.js

```diff
@@ -101,3 +103,29 @@ Child ${config2Name}:
 		expect(hashInfo.config[1]).toEqual({ name: config2Name, hash: config2Hash });
 	});
 });
+
+describe("appendFile functionality", () => {
+	describe("positive test-cases", () => {
+		const junkFile = resolve(__dirname, "junkFile.js");
+
+		beforeEach(() => {
+			writeFileSync(junkFile, "");
```

Comment:
Here you're writing nothing an magically a junk comment appears if you see the analogy. Write something in `beforeEach` and then run `expect` after before each, then you need to make a change and check that it actually changes content as well

Reply:
Ok. Then you need to assert(expect) ∆data from the appending step (2) and number (4) and make sure that they hold.